runtime/CSharp/runtime/CSharp/Antlr4.Runtime/ParserRuleContext.cs

```diff
@@ -252,7 +252,7 @@ public virtual ITerminalNode GetToken(int ttype, int i)
             return null;
         }
 
-#if NET45PLUS
+#if (NET45PLUS && !DOTNETCORE)
```

Comment:
have you tried dropping NET45PLUS altogether (in the entire project)? I think I only kept it to avoid trashing the deactivated code.